tools/isobuild/linker.js

```diff
@@ -956,9 +956,8 @@ var getHeader = function (options) {
   });
 
   chunks.push(
-      `Package["core-runtime"].queue("${options.name}", [`,
-      orderedDeps.join(', '),
-      '], function () {'
+      `Package["core-runtime"].queue("${options.name}",`,
```

Comment:
You should be able to completely remove the `deps` option from fullLink, and replace it with a `hasRuntime` option instead. This would also help with the linker cache so it isn't invalidated as often.